.github/workflows/e2e.yml

```diff
@@ -107,15 +108,22 @@ jobs:
           # Check the installed version of @playwright/test
           echo "@playwright/test version: $(npm list @playwright/test --depth=0)"
 
+      - name: Get Template Version
+        run: |-
+          version=`jq -r ".version" packages/template-chakra-storefront/package.json`
+          echo "retail_app_template_version=$version" >> "$GITHUB_ENV"
+
       - name: Generate Retail App Without Extensibility
         uses: ./.github/actions/e2e_generate_app
         with:
           PROJECT_KEY: "retail-app-no-ext"
+          TEMPLATE_VERSION: ${{ env.retail_app_template_version }}
 
       - name: Validate Retail App Without Extensibility
         uses: ./.github/actions/e2e_validate_generated_app
         with:
           PROJECT_KEY: "retail-app-no-ext"
+          TEMPLATE_VERSION: ${{ env.retail_app_template_version }}
```

Comment:
What is retail-app-ext and retail-app-no-ext? How do they translate to the new structure?

Reply:
Yeah, let's get rid of this. Will fix